website/blog/2023-12-06-0.73-debugging-improvements-stable-symlinks.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'React Native 0.73 - Debugging Improvements, Stable Symlink Support, and more'
+authors: [fortmarek, titozzz, hurali, lunaleaps, huntie]
+tags: [announcement, release, debugging]
+date: 2023-12-06
+---
+
+Today we're releasing React Native 0.73! This release adds improvements to debugging with Hermes, stable symlink support, Android 14 support, and new experimental features. We are also deprecating legacy debugging features. We continue to make foundational improvements in preparation for the New Architecture.
```

Comment:
Bridgeless mode is a very important milestone for the new architecture rollout in open source. We should try to get more people using and adopting bridgeless mode in 0.73. So, let's not hide it under "foundational improvements" or "New Architecture updates", and highlight it from the get-go so we get peoples' eyes on it.

```suggestion
Today we're releasing React Native 0.73! This release adds improvements to debugging with Hermes, stable symlink support, Android 14 support, and new experimental features. We are also deprecating legacy debugging features. We are also releasing the next pillar of the New Architecture: Bridgeless Mode!
```

Reply:
@huntie, how do you feel about moving this higher up in the list, also?